input/params/simulations/Genesis_L210_N4320.par

```diff
@@ -27,3 +27,35 @@ NPart: 80621568000
 UnitLength_in_cm         : 3.08568e+24       # WATCH OUT : Mpc/h
 UnitMass_in_g            : 1.989e+43         # WATCH OUT : 10^10Msun/h
 UnitVelocity_in_cm_per_s : 100000            # WATCH OUT : km/s
+
+
+#-------------------------------------------------
+#----- Astrophysical parameters since Balu+22 ----
+#-------------------------------------------------
+
+ReionNionPhotPerBary : 6000
+ReionGridDim : 1024
+
+ReincorporationModel : 2
+ReincorporationEff        : 18000    # gas reincorporation efficiency from ejected component
+
+SfEfficiency              : 0.09966
+SfCriticalSDNorm          : 0.01382
+
+SnModel                   : 2
+SnEjectionEff             : 1.53782     # SN feedback gas reheating efficiency
+SnEjectionScaling         : 1.0
+SnEjectionScaling2        : 3.2
+
+SnReheatRedshiftDep       : 2.0
+SnReheatEff               : 6.97996    # mass loading factor for SN feedback
+SnReheatScaling           : 1.0
+SnReheatScaling2          : 3.2
+SnReheatNorm              : 60.0
+SnReheatLimit             : 10000.0    # maximum allowed mass loading factor for SN feedback
+
+EscapeFracDependency : 5 # 0->none, 1->redshift, 2->stellar mass, 3->sfr, 4->cold gas surface density, 5->mvir, 6->ssfr
+EscapeFracNorm : 0.15
+EscapeFracPropScaling : 0.00
+EscapeFracRedshiftScaling : 0
+EscapeFracBHNorm : 0
```

Comment:
i've decided to keep these changes in the simulation parameters **for now**.  
@s-balu  can you confirm these are what you were using?
@EMventura are you also using these in your L10 boxes? if so, these need to be included in L10.par too.

Reply:
What if, instead of using the L10 parameters as the default we don't include those parameters in the default.par and just split those in the L10 vs L210?

Reply:
I think the current ones in default.par are used by tiamat trees. If we don't have them, we need to modify every file in simulations.

Reply:
What i was suggesting is to include these parameters in l10.par If they are different from those in default.par

Reply:
The one I used for the L10 box are the ones in default.par as they were inherited from TinyTiamat which was the simulation with the more similar resolution compared to the Genesis L10N2048.

Reply:
okay, then there is no need to change its parameter file. 

By the way, does this explain the difference you saw in Balu and your models running on the same trees? Did you use different parameter files by any chance?

Reply:
No when I ran on the L210 box I change all those parameters (we both double checked with a diff in the log files and indeed we are using the same parameters).

Reply:
okay, let's wait for @s-balu 's confirmation before we merge it

Reply:
Hi @qyx268 and @EMventura,
sorry about the delay in getting back about this.

My escape fraction prescription is different. I was using the one that depends only on redshift:

`EscapeFracDependency      : 1  # 0->none, 1->redshift, 2->stellar mass, 3->sfr, 4->cold gas surface density, 5->mvir, 6->ssfr
EscapeFracNorm            : 0.14
EscapeFracRedshiftOffset  : 6.0
EscapeFracRedshiftScaling : 0.2`

Reply:
This might be true, I took these from a suite of runs where I explore EscapeFracDependency:5.

Just to confirm, you meant to add these in Genesis_L210_N4320.par? i.e. apart from the escape fraction parameters, the following astro parameters are what you were using.
Plus, removing `EscapeFracBHNorm : 0` meanings you were considering AGN ionization contribution? 

```
#-------------------------------------------------
#----- Astrophysical parameters since Balu+22 ----
#-------------------------------------------------

ReionNionPhotPerBary : 6000
ReionGridDim : 1024

ReincorporationModel : 2
ReincorporationEff        : 18000    # gas reincorporation efficiency from ejected component

SfEfficiency              : 0.09966
SfCriticalSDNorm          : 0.01382

SnModel                   : 2
SnEjectionEff             : 1.53782     # SN feedback gas reheating efficiency
SnEjectionScaling         : 1.0
SnEjectionScaling2        : 3.2

SnReheatRedshiftDep       : 2.0
SnReheatEff               : 6.97996    # mass loading factor for SN feedback
SnReheatScaling           : 1.0
SnReheatScaling2          : 3.2
SnReheatNorm              : 60.0
SnReheatLimit             : 10000.0    # maximum allowed mass loading factor for SN feedback

EscapeFracDependency : 1 # 0->none, 1->redshift, 2->stellar mass, 3->sfr, 4->cold gas surface density, 5->mvir, 6->ssfr
EscapeFracNorm : 0.14
EscapeFracRedshiftOffset : 6.0
EscapeFracRedshiftScaling : 0.2
```

Reply:
@qyx268 yes that is correct. I had AGN feedback on. 
https://gist.github.com/s-balu/5c19459b63ea6d16ce73c61ffbca54aa

This is the relevant sections from the input par I used